packages/cloud_functions/cloud_functions_platform_interface/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017-2020 The Chromium Authors. All rights reserved.
```

Comment:
I am not a lawyer but I think it would be best if this LICENSE file copied directly from the one in `packages/cloud_functions/cloud_functions/LICENSE` and had the Copyright year set to 2018

Reply:
Because of the way copyright works, one wants to claim copyright as soon as a work is published and then restate the claim any time new work is performed. Otherwise, the copyright will lapse early.

https://epiphany.law/articles/copyright/copyright-dates-single-year-or-range/